src/components/Greeting.jsx

```diff
@@ -0,0 +1,15 @@
+import { useSelector } from 'react-redux';
+
+const Greeting = () => {
+  let greetings = useSelector((state) => state.greetings.greetings);
+  const status = useSelector((state) => state.greetings.status);
+  const randomIndex = Math.floor(Math.random() * greetings.length);
+  console.log(status);
+  console.log(greetings);
+
```

Comment:
- [ ] 👍 Great job on incorporating the random greeting feature on the front end! 😊 However, please keep in mind that the API endpoint is expected to provide a random greeting. You can achieve this by defining the index action in your GreetingsController to return a random greeting, just like this:
```
class GreetingsController < ApplicationController
  def index
    @random_greeting = Greeting.order("RANDOM()").first
    render json: { greeting: @random_greeting&.content }
  end
end

```